TDAT: public void test delete user failure ( ) throws remote exception {  <NL> string handle = _ gateway . connect ( " admin " , " yawl " ) ;  <NL> assert not null ( handle ) ;  <NL>  <NL>  / / delete the user <NL> string result = _ gateway . delete user ( " admin " , handle ) ;  <NL> assert not null ( result ) ;  <NL> assert true ( result , result . starts with ( " < failure " ) ) ;  <NL>  }  COM: <s> delete user should fail if you try to delete yourself </s>